src/DynamoCoreWpf/ViewModels/Core/DynamoViewModelBranding.cs

```diff
@@ -1,22 +1,9 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Windows.Media;
-using Dynamo.Wpf.Interfaces;
+﻿using Dynamo.Wpf.Interfaces;
 
 namespace Dynamo.ViewModels
 {
     partial class DynamoViewModel 
     {
         public IBrandingResourceProvider BrandingResourceProvider { get; private set; }
-
-        public ImageSource AboutBoxIcon
-        {
-            get
-            {
-                return BrandingResourceProvider.GetImageSource(ResourceNames.AboutBox.Image);
-            }
-        }
```

Comment:
I had thought of the same but I felt like just leaving it in here just in case there are other branding specific properties/methods we may need to add to this file in the near future.